jcl/src/openj9.sharedclasses/share/classes/com/ibm/oti/shared/SharedClassUtilities.java

```diff
@@ -107,8 +108,16 @@
 	 * 					If shared classes is disabled for this JVM (that is -Xshareclasses:none is present).
 	 * @throws		IllegalArgumentException
 	 * 					If <code>flags</code> is not a valid value.
+	 * @throws		SecurityException
+	 * 					If a security manager is enabled and the calling thread does not
+	 * 					have SharedClassesUtilPermission("getSharedCacheInfo")
```

Comment:
The permission class is `SharedClassesNamedPermission`, right? Please update the javadocs (here and in `destroySharedCache`).